pytest_watcher/watcher.py

```diff
@@ -50,16 +87,19 @@ class EventHandler(events.FileSystemEventHandler):
         events.EVENT_TYPE_MOVED,
     )
 
+    def __init__(self, file_filter: Optional[FileFilter] = None):
```

Comment:
I don't think it's a good idea to overwrite `__init__` in a class that we don't own. This creates additional maintenance overhead in case the original `__init__` will change in the `watchdog` library.